service/src/test/resources/ehr/mapper/referral/expected-output-request-statement-9.xml

```diff
@@ -6,6 +6,6 @@
         </effectiveTime>
         <availabilityTime value="20100119"/>
         <code code="3457005" displayName="Patient referral" codeSystem="2.16.840.1.113883.2.1.3.2.4.15"/>
-        <text>UBRN: bb2378b6-dde2-11e9-9d36-2a2ae2dbcce4 Priority: Routine Requester: Patient Specialty: unknown Referred for investigation into low blood pressure</text>
+        <text>UBRN: bb2378b6-dde2-11e9-9d36-2a2ae2dbcce4 Priority: Routine Specialty: unknown Referred for investigation into low blood pressure</text>
```

Comment:
looking at data and I think that case 9 should have` Requester: Patient` still, it has in ReferralRequest:

```
  "requester": {
        "agent": {
            "reference": "Patient/4C903809-CADA-442E-8FA5-655E5EFEB1F5"
        }
    }```

Reply:
It doesn't because the input fhir-bundle.json doesn't have this. I'll add it to the bundle